Content.Server/StationEvents/Components/VentScrubberSpawnLocationComponent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Content.Server.StationEvents.Events;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent]
+public sealed class VentScrubberSpawnLocationComponent : Component
+{
+
+}
```

Comment:
Component name needs to be more generic, e.g. "MobSpawnLocationComponent" and also needs commenting.

Reply:
`MobScrubberSpawnLocationComponent`
It is still worth specifying in the component that it uses scrubber as a spawn point?